suggester/src/main/java/org/opengrok/suggest/SuggesterProjectData.java

```diff
@@ -261,8 +261,10 @@ private WFSTCompletionLookup build(final IndexReader indexReader, final String f
         WFSTCompletionLookup lookup = createWFST();
         lookup.build(iterator);
 
-        double averageLength = (double) iterator.termLengthAccumulator / lookup.getCount();
-        averageLengths.put(field, averageLength);
+        if (iterator.termLengthAccumulator > 0) {
```

Comment:
I'd prefer the condition to check `lookup.getCount()` but that's not a big deal.